anitya/lib/backends/__init__.py

```diff
@@ -79,11 +81,11 @@ class BaseBackend(object):
             checking for new versions. This could be overriden by backend plugin.
     """
 
-    name = None
-    examples = None
-    default_regex = None
-    more_info = None
-    default_version_scheme = None
+    name : str
+    examples : List[str]
+    default_regex = REGEX
```

Comment:
This should be Optional, because the REGEX is only used for custom backend. I would go with str instead.